aio/develop/Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN curl -sL https://deb.nodesource.com/setup_11.x | bash - \
 	&& apt-get clean
 
 # Install dependencies. This will take a while.
-RUN npm install -g npm@latest gulp
+RUN npm install -g npm@latest gulp @angular/cli@latest
```

Comment:
`latest` can be problematic with the new releases. Let's keep that in mind for the future.

Reply:
Thanks! Versions for gulp and angular/cli were specified same as written in package.json in #3971